x-pack/test/functional/fixtures/kbn_archiver/spaces/disabled_features.json

```diff
@@ -0,0 +1,16 @@
+    "timeFieldName": "@timestamp",
+    "title": "logstash-*"
+  },
+  "coreMigrationVersion": "8.0.0",
```

Comment:
This should be exported from `7.x` so it can be cleanly backported.

Reply:
Does that mean simply changing the value to `7.x`, or rebasing off of the `7.x` branch, or something else?  Oh and thanks for reviewing mate :)

Reply:
You can change it and see if it works, but we really should get in the habit of creating the export from 7.x in the first place.

Reply:
@tylersmalley ah I see, gotcha.  I'll do that.

Reply:
Done, pushing soon.